lisa/sut_orchestrator/azure/common.py

```diff
@@ -1630,6 +1630,20 @@ def get_deployable_vhd_path(
     full_vhd_path = copy_vhd_to_storage(
         platform, storage_name, original_vhd_path, vhd_path, log
     )
+    log.debug(f"deployable vhd path: {full_vhd_path}")
```

Comment:
It should have the original vhd path in the same line, so it's easy to look for, like "deployable vhd is cached at: XXX, original vhd url: xxx."